tools/db_stress_tool.cc

```diff
@@ -617,6 +617,11 @@ DEFINE_string(checksum_type, "kCRC32c", "Algorithm to use to checksum blocks");
 static enum rocksdb::ChecksumType FLAGS_checksum_type_e = rocksdb::kCRC32c;
 
 DEFINE_string(hdfs, "", "Name of hdfs environment");
+
+DEFINE_string(env_uri, "",
```

Comment:
`WITH_TOOLS` option is ON at the end of main cmake process, and I got error like `/path/to/db_stress.cc:20: undefined reference to 'rocksdb::db_stress_tool(int, char**)'`.

https://github.com/facebook/rocksdb/blob/f65ec09ef86c8dffd20aa63a12126a23d6d3c5b4/CMakeLists.txt#L1121

While trying to add `tools/db_stress_tool.cc` to `SOURCES` afterwards, seems something conflicts like `/path、to/db_stress_tool.cc:1817: multiple definition of 'fLS::FLAGS_env_uri'`.

Everything goes fine before, since codes was put in `db_stress.cc`. Can we try to merge common stuff together, or any solutions?